modules/transactions/src/main/java/org/apache/ignite/internal/tx/storage/state/rocksdb/TxStateRocksDbStorage.java

```diff
@@ -398,28 +391,31 @@ public void destroy() {
     }
 
     private byte[] partitionStartPrefix() {
-        return ByteBuffer.allocate(Short.BYTES).order(ByteOrder.BIG_ENDIAN)
-            .putShort((short) (partitionId))
-            .array();
+        return ByteBuffer.allocate(PREFIX_SIZE_BYTES).order(ByteOrder.BIG_ENDIAN)
```

Comment:
Maybe byte order to constant or static import?

Reply:
I'll do a static import